### PR TITLE
Add instructions how to build for MacOS machines

### PR DESCRIPTION
**Pull request by scout719**

### CODE REVIEW

File: BUILD_FOR_MAC.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    * `install_name_tool -change $HOME/build_ios_proxy/install_folder/lib/libcrypto.1.1.dylib @executable_path/libcrypto.1.1.dylib ios_webkit_debug_proxy`
+    * `install_name_tool -change $HOME/build_ios_proxy/install_folder/lib/libssl.1.1.dylib @executable_path/libssl.1.1.dylib libimobiledevice-1.0.6.dylib`
+    * `install_name_tool -change $HOME/build_ios_proxy/install_folder/lib/libcrypto.1.1.dylib @executable_path/libcrypto.1.1.dylib libimobiledevice-1.0.6.dylib`
+    * `install_name_tool -change $HOME/build_ios_proxy/install_folder/lib/libusbmuxd-2.0.6.dylib @executable_path/libusbmuxd-2.0.6.dylib libimobiledevice-1.0.6.dylib`
+    * `install_name_tool -change $HOME/build_ios_proxy/install_folder/lib/libplist-2.0.3.dylib @executable_path/libplist-2.0.3.dylib libimobiledevice-1.0.6.dylib`
+    * `install_name_tool -change $HOME/build_ios_proxy/install_folder/lib/libcrypto.1.1.dylib @executable_path/libcrypto.1.1.dylib libssl.1.1.dylib`
+    * `install_name_tool -change $HOME/build_ios_proxy/install_folder/lib/libplist-2.0.3.dylib @executable_path/libplist-2.0.3.dylib libusbmuxd-2.0.6.dylib`
+
+---
```
Comment (bgracio) [inline BUILD_FOR_MAC.md]: Small note, remove `---`  😄

**Review by bgracio — DISMISSED** [review]

**Review by SaraAGoncalves — DISMISSED** [review]
So many steps 😱
Food for thought: couldn't we automate all this with some build script? After reading the notes I also had the question when do I actually need to do this? 😅

**Comment by scout719** [comment]
> So many steps 😱
> Food for thought: couldn't we automate all this with some build script? After reading the notes 

I think it should be possible, but given the frequency that we need to generate this binaries, it's not worth the effort

> I also had the question when do I actually need to do this? 😅

There is a file on the Main repo, with a pointer to this guide so that people find it when they need to update the binaries shipped with the platform

File: BUILD_FOR_MAC.md (hunk 1 above)
Comment (scout719) [reply BUILD_FOR_MAC.md]: But it's so pretty 😂

**Review by bgracio — APPROVED** [review]

**Pull request merged**